### ISSUE TITLE
Issue #12 Director's Dashboard - Registration Report Priority=URGENT

### ISSUE THREAD

**Issue by kirkwhiteman1** [YBNetwork/Issues-Enhancements#12]
Roopesh coded the ability to e-mail teams from the registration report 3 ways, and I know the e-mails are NOT getting to the teams.

1. Director clicks the mail icon - 
2. Director click's VIEW TEAM eyeball icon.  Popup then there is another e-mail icon.  Doesn't work
3. EMAIL Group Button top right.  User clicks this - Popup is below.  Teams are not getting these e-mails either.
![image](https://cloud.githubusercontent.com/assets/28075903/25617441/eee0802c-2f07-11e7-92e3-837295d41461.png)

Need this to be fixed so teams receive e-mails.

Thanks,

Kirk

**Comment by kirkwhiteman1** [comment]
Issue #12 and Issue #13 are both URGENT and getting to the NOW priority level based on feedback from my directors using the site.  I would say ISSUE #12, fixing the e-mail should be 1st then Issue #13.  I have no way to get the e-mail list to my directors right now and they need to communicate with their teams.

Please advise.  KIRK

**Comment by bguerrettaz** [comment]
Ok, I'm in the middle of #13.  I'll move to #12 now and then finish #13
after that.

I'll turn it around as quickly as possible.

Blake

**Comment by kirkwhiteman1** [comment]
I forgot to add.  Is there anyway you can allow attachments to be uploaded and sent when fixing this issue?

**Comment by bguerrettaz** [comment]
Kirk - I worked on this most of today/this evening so you'd have it up and running.  I think I have it working correctly for both individual and group emails.

Can you test all three spots to assure that it is emailing correctly and as you'd expect?

If so, we can leave this open and you can let me know what priority adding attachments is, since that will take some time as well.

Blake

**Comment by kirkwhiteman1** [comment]
I tested all 3. Looks good. When I tested the "EMAIL GROUP" I selected my Spring Tournament May 27. It took a while to "send" about 15 seconds. So I hit the button again and it immediately sent, then sent again. I received two of these e-mails. It's functional. Thanks for the quick turnaround.

 

I think I put this reply under ISSUE #15 – just so you know.  

 

Yours in Hoops,

 

Kirk Whiteman

217.714.7263

 <http://www.aybtour.com/> 

 <http://www.ybnetwork.net/>      <https://www.facebook.com/americanyouthbasketballtour/>      <https://twitter.com/AybTour>      <https://www.instagram.com/aybtour/>

**Comment by kirkwhiteman1** [comment]
I tested all 3. Looks good. When I tested the "EMAIL GROUP" I selected my Spring Tournament May 27. It took a while to "send" about 15 seconds. So I hit the button again and it immediately sent, then sent again. I received two of these e-mails. It's functional. Thanks for the quick turnaround.

I think I put this reply under ISSUE #15 – just so you know.

**Comment by bguerrettaz** [comment]
Yes, I was worried about the slowness of the group email send, the code that was written was horrific, my goal was to get it functional for you, but I would advise that we come back and rewrite a portion of this to make it perform better.  Thoughts?

Is this connected to #15 as well?

**Comment by kirkwhiteman1** [comment]
No not connected to 15.

Just had a director text.  He tried the group email function for Paid
Teams..said nothing happened.  Doug Nichols wgo is running event this
weekemd

Told him had to wait a few seconds

Kirk Whiteman

**Comment by bguerrettaz** [comment]
Is this issue ready to close, or is there something I should take a look at?

**Comment by bguerrettaz** [comment]
I want to make sure all of the URGENT priority tasks are good to go.

Is this ready to be close?

**Comment by kirkwhiteman1** [comment]
I think we need to keep this open and revisit end of summer as it seems very SLOW to send the group e-mail?  I tried it again just now and it is VERY Slow.

**Comment by bguerrettaz** [comment]
Sounds good, thanks!
